### PR TITLE
Recipe page

### PR DESCRIPTION
**Pull request by MBHunter88**
Fixed spelling and grammar typos, updated CSS and added media queries and a light/dark mode toggle feature. Also, deleted "search" input on navbar. 

The result is a recipe web-page that is interactive and responsive on different devices.

### CODE REVIEW

File: techtonica-projects/recipe-page/styles.css [hunk 1]
```diff
@@ -129,6 +129,7 @@ h1:hover {
 @media (min-width: 1025px) {
   h1 {
       font-size: 2em;
+      /* test color: green; */
```
Comment (Shikha2410) [inline techtonica-projects/recipe-page/styles.css:148]: I am assuming this commit is to convey you've attempted media queries testing.

**Comment by Shikha2410** [comment]
Well done on providing the description of changes in your PR, helps me in making sense of the modifications to code. 
I would love for you to attach a visual artifact next time- maybe a screenshot of the website which reflects the before/after states, or something like that.

**Review by Shikha2410 — APPROVED** [review]
Good to go into main!

File: techtonica-projects/recipe-page/styles.css (hunk 1 above)
Comment (MBHunter88) [reply techtonica-projects/recipe-page/styles.css:148]: Yes because the font size change was so minimal I wanted to see the change more clearly.

**Pull request merged**